### ISSUE TITLE
when can i use the new Erlang page

### ISSUE THREAD

**Issue by LosedMask** [erlang/erlide_eclipse#333]
I notice that there is a new page call new erlang in Erlide but it's not work anymore.  It's too complex to add every single folder in Erlang page by hand. if it can be chosen, it will be great.

**Comment by vladdu** [comment]
Hi! As you may have noticed from the commit history, I've not been active for erlide lately. I will see what I can do, but I can't promise it will be soon. Currently that properties page has no functionality behind it. 

I guess you are not using standard project structure?

**Comment by LosedMask** [comment]
> Hi! As you may have noticed from the commit history, I've not been active for erlide lately. I will see what I can do, but I can't promise it will be soon. Currently that properties page has no functionality behind it.
> 
> I guess you are not using standard project structure?

thx.    


Yes, i am not using standard project structure.   :(  so it's sad that i have to add the folder one by one. it's a bit of  a waste of time.

**Comment by vladdu** [comment]
I agree that it is boring, but it is only needed once per project. You can also edit the .settings/org.erlide.core.prefs directly, if that makes it easier.
